modules/dataquery/jsx/react.app.js

```diff
@@ -259,21 +263,31 @@ class DataQueryApp extends Component {
           queryIDs.User.push(id);
         }
       }
-      $.get(loris.BaseURL + '/AjaxHelper.php?Module=dataquery&script=GetDoc.php&DocID=' + id,
-        (value) => {
-          let queries = this.state.savedQueries;
+      fetch(
+        window.location.origin
```

Comment:
same here

Reply:
@HenriRabalais the use of the dollar sign here isn't jQuery. See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals